### ISSUE TITLE
TypeError: goog.isString is not a function

### ISSUE THREAD

**Issue by thomthom** [google/closure-templates#106]
I ran into an issue where when I use Closure Templates standalone I get the following error: `test.js:20 Uncaught TypeError: goog.isString is not a function`

<details>
<summary>test.html</summary>
```
<html>
<head>
  <title>The Hello World of Closure Templates</title>
  <script type="text/javascript" src="soyutils.js"></script>
  <script type="text/javascript" src="test.js"></script>
</head>
<body>
  <script type="text/javascript">
    document.write(examples.list());
  </script>
</body>
</html>
```
</details>

**Comment by thomthom** [comment]
I added the following snipped in a separate js file that I loaded and that worked around the `isString` error:

```
/**
 * Returns true if the specified value is a string.
 * @param {?} val Variable to test.
 * @return {boolean} Whether variable is a string.
 */
if (!goog.isString) {
  goog.isString = function(val) {
    return typeof val == 'string';
  };
}
```

But I then ran into a new error:

```
Uncaught TypeError: goog.asserts.assertNumber is not a function
```

Is there a different way to do this? Or is it a bug?

**Comment by mikesamuel** [comment]
Where did you get your `soyutils.js`?

**Comment by thomthom** [comment]
From the `gulp-soynode` npm module. I just compared it to the zip file from this page: https://developers.google.com/closure/templates/docs/helloworld_js
...and I see they are different.

It looks like the `gulp-soynode` module makes use of `closure-templates`: https://www.npmjs.com/package/closure-templates

I has seen the `closure-templates` folder in the `node_modules` folder and assumed that was upto date - and set up[ my gulp project to copy that file.

I will try to swap out with the one from the Closure Templates website and see how that goes.

**Comment by mikesamuel** [comment]
https://github.com/google/closure-templates/commit/169d5b015ef67418cd70cbf55549068c81eb6197 may address this, though I'd still like to hear whether a recent soyutils.js fixes this.

**Comment by thomthom** [comment]
Yes - I'll report back. I have been delayed in testing a newer `soyutils.js` because I had to prioritise some urget work. I should be able to get to it next week.

**Comment by marisev** [comment]
Hi,

I'm also running into "goog.isString is not a function" error. Just downloaded the most recent zip from https://developers.google.com/closure/templates/docs/helloworld_js . Trying to declare a parameter inside template:
```
{template .Main}
 {@param testTxt: string} 
 {$testTxt}
{/template}
```
Any advice? Should I download "soyutils.js" from github source?

Thanks!
